db/Cargo.toml

```diff
@@ -13,8 +13,12 @@ ckb-app-config = { path = "../util/app-config", version = "= 0.104.0-pre" }
 ckb-logger = { path = "../util/logger", version = "= 0.104.0-pre" }
 ckb-error = { path = "../error", version = "= 0.104.0-pre" }
 libc = "0.2"
-rocksdb = { package = "ckb-rocksdb", version ="=0.16.1", features = ["snappy"] }
+rocksdb = { package = "ckb-rocksdb", version ="=0.18.0", features = ["snappy"], default-features = false }
```

Comment:
Please upgrade rocksdb in a separate PR.

Reply:
https://github.com/nervosnetwork/ckb/pull/3468